cpp/velox/utils/VeloxArrowUtils.cc

```diff
@@ -36,7 +36,7 @@ arrow::Status MyMemoryPool::Allocate(int64_t size, int64_t alignment, uint8_t**
   if (bytes_allocated() + size > capacity_) {
     return arrow::Status::OutOfMemory("malloc of size ", size, " failed");
   }
-  RETURN_NOT_OK(pool_->Allocate(size, out));
+  RETURN_NOT_OK(pool_->Allocate(size, alignment, out));
```

Comment:
need to do an aligned allocate, is this also related with the leakage?

Reply:
No... It's just some code looking buggy so I fixed in passing. The code is only used for testing afaik.